test/ImpactEvaluator.t.sol

```diff
@@ -691,4 +822,21 @@ contract ImpactEvaluatorTest is Test {
 
         assertEq(vm.addr(1).balance, 0);
     }
+
+    function test_ParticipantIsReadyForTransfer() public {
+        ImpactEvaluator impactEvaluator = new ImpactEvaluator(address(this));
+        address payable[] memory addresses = new address payable[](2);
+        addresses[0] = payable(vm.addr(1));
+        addresses[1] = payable(vm.addr(2));
+        uint[] memory balances = new uint[](2);
+        balances[0] = 50 ether;
+        balances[1] = 50 ether;
+        impactEvaluator.addBalances{value: 100 ether}(addresses, balances);
+
+        assert(impactEvaluator.participantIsReadyForTransfer(addresses[0]));
+        assert(impactEvaluator.participantIsReadyForTransfer(addresses[1]));
+        assert(
+            !impactEvaluator.participantIsReadyForTransfer(payable(vm.addr(3)))
+        );
```

Comment:
What is the benefit of testing two addresses with the same balance? Isn't it enough to test this logic for one address only?

The last assertion checks the case when the address does not have any balance, which makes sense.

I am missing a check for the case when the address has some balance, but that balance is lower than the threshold for transfer.

Reply:
good points, I didn't think this one through enough. Fixed